dietpi/dietpi-software

```diff
@@ -1305,6 +1305,14 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_DESC[$software_id]='prevents brute-force attacks with ip ban'
 		aSOFTWARE_CATX[$software_id]=11
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/system_security/#fail2ban'
+		#------------------
+		software_id=171
+
+		aSOFTWARE_NAME[$software_id]='frp'
+		aSOFTWARE_DESC[$software_id]='reverse proxy'
+		aSOFTWARE_INTERACTIVE[$software_id]=1
+		aSOFTWARE_CATX[$software_id]=11
+		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/system_security/#frp'
 
```

Comment:
Should it not be better localized in [Advanced Networking](https://dietpi.com/docs/software/advanced_networking), i.e. together with HAProxy?

Reply:
I agree. It focuses on filtering/firewalling features, but every proxy and webserver can do this, when configured accordingly. A proxy on it's own is never a security feature, but in the first place it openes another access way and can only make this additional path secure.